apps/web/playwright/organization/booking-list.e2e.ts

```diff
@@ -0,0 +1,407 @@
+import type { Page } from "@playwright/test";
+import { expect } from "@playwright/test";
+
+import { getOrgFullOrigin } from "@calcom/features/ee/organizations/lib/orgDomains";
+import { randomString } from "@calcom/lib/random";
+import slugify from "@calcom/lib/slugify";
+import prisma from "@calcom/prisma";
+import { MembershipRole, SchedulingType } from "@calcom/prisma/enums";
+
+import { createTeamEventType } from "../fixtures/users";
+import { test } from "../lib/fixtures";
+import type { Fixtures } from "../lib/fixtures";
+import {
+  assertBookingVisibleFor,
+  bookTimeSlot,
+  doOnOrgDomain,
+  NotFoundPageTextAppDir,
+  selectFirstAvailableTimeSlotNextMonth,
+  testName,
+} from "../lib/testUtils";
+
+test.describe("Bookings list for organizations", () => {
+  test.afterEach(({ orgs, users }) => {
+    orgs.deleteAll();
+    users.deleteAll();
+  });
+
+  test.describe("Team Event", () => {
+    test("Collective EventType booking list", async ({ page, users, orgs }) => {
+      const { org, orgOwner, orgAdmin, team1owner, team2owner, commonUser, team1_teammate1 } =
+        await createOrg(orgs, users);
+
+      const { team: team1 } = await team1owner.getFirstTeamMembership();
+      const scenario = {
+        schedulingType: SchedulingType.COLLECTIVE,
+        teamEventTitle: `collective-team-event`,
+        teamEventSlug: slugify(`collective-team-event-${randomString(5)}`),
+      };
+      const eventType = await createTeamEventType(team1owner, team1, scenario);
+      const { id: eventId, slug: evnetSlug } = eventType;
+
+      await prisma.host.createMany({
+        data: [
+          {
+            userId: commonUser.id,
+            eventTypeId: eventId,
+            isFixed: true,
+          },
+          {
+            userId: team1_teammate1.id,
+            eventTypeId: eventId,
+            isFixed: true,
+          },
+        ],
+      });
+
+      // to test if booking is visible to team admin/owner even if he is not part of the booking
+      await prisma.host.deleteMany({
+        where: {
+          userId: team1owner.id,
+          eventTypeId: eventId,
+        },
+      });
+      await expectPageToBeNotFound({ page, url: `/team/${team1.slug}/${eventType.slug}` });
+      await doOnOrgDomain(
```

Comment:
can someone help me please,
i was writing tests for booking-list in org context but, i'm having trouble running org tests locally
this function always get me `page does not exist`
tried running other test this function was called, same result
is there a different way to run org tests locally

Reply:
@SomayChauhan can you try the following:
> You need to make sure that you run the dev server using this
> NEXT_PUBLIC_IS_E2E=1 yarn dev
> It will fake the org domain on a regular domain

Reply:
@supalarry works!!
thankyou